src/keyszer/config_api.py

```diff
@@ -198,7 +224,9 @@ def unicode_keystrokes(n):
             for digit in _digits(n, 16)
             for hexdigit in hex(digit)[2:].upper()
             ],
-        Key.ENTER
+        sleep_ms(_THROTTLES['unicode_delay_ms']),
+        Key.ENTER,
+        sleep_ms(_THROTTLES['unicode_delay_ms']),
     ]
```

Comment:
This doesn't do anything other than sleep a little while the config first runs generating unicode sequences... these would need to be commands (that return lambdas that sleep) for them to have any runtime impact.  If you think this has been changing behavior at runtime then you're seeing placebo effects...

Just realized when I saw your sleep helper was in the config API...

Reply:
🤦🏽 Ah, that is quite correct. I hadn't noticed it doing much lately, but I've only been putting 10ms as a delay there, if anything. That explains a lot. 

So that needs to be something that would have an effect in config, in other words the original sleep helper function. Or something similar that returns a function that will do the job. 🙀 I'll figure it out.

Reply:
Honest assessment of the Unicode delay: It really isn't very effective, even when it's changed back to using the `sleep()` helper so that it actually works. It's the other keystroke delay that fixes things like the Shift-Ctrl-u shortcut being misinterpreted. 

Although I can say for certain that the `sleep(0.1)` just before the Enter key did fix the Unicode sequences in Kubuntu. That was a real phenomenon that I witnessed. But in that case I don't think I did much if any testing of other macros. There probably would be extensive problems with macros in general to go along with the Unicode failures. I just didn't spend much time in Kubuntu, so I only noticed right away that my special characters were consistently failing. 

I hate to say it, but it would probably be OK to entirely remove the Unicode delay knob and the attempt at sleeping around the Enter key. If this timing problem is present, the keystroke delay around all non-modifier keys should be all that will be necessary to also fix Unicode sequences. Because that will insert a delay before and/or after the Enter key, and will also far more effectively fix the original Shift-Ctrl-u shortcut that initiates the Unicode sequence. 

Cranking the Unicode knob down to zero really has no negative effect, as long as the other delay knobs are sufficiently high to keep bad things from happening in macros in general. 

I'd be fine with removing that knob and just relying on the pre/post keystroke delays. Avoid wasting the user's time playing with a knob that probably does nothing that helpful, except by coincidence in very limited circumstances. 

You good with that?

Reply:
I've done enough testing in my Kubuntu VM now to confirm that it doesn't just have the Unicode issue. Once you try a complicated macro full of shifted characters and Unicode characters, it needs pretty much the same solution as the Ubuntu VM for everything to come out without problems. Which is about 50-70 milliseconds on either side of the "normal" keystroke in `send_combo`. 

I made sure to re-enable the sync line that I had disabled as an alternative "solution" to the sleep delay in the Unicode function. So there was no longer any Unicode delay, and there were numerous macro defects until adding a nearly identical set of keystroke delays that I've been working with for days in the other VM. 

I'm going to remove the Unicode delay. I don't think there's good evidence anymore that it serves a real purpose independently of the primary keystroke delays.